src/main/kotlin/io/perfometer/dsl/HttpDsl.kt

```diff
@@ -105,18 +111,19 @@ class Scenario(
         private val baseURL: URL,
         private val builder: suspend HttpDsl.() -> Unit,
 ) {
-    private var runner: ScenarioRunner = CoroutinesScenarioRunner(SimpleHttpClient(false))
+    private var runner: ScenarioRunner = CoroutinesScenarioRunner(KtorHttpClient())
 
     fun runner(runner: ScenarioRunner): Scenario {
         this.runner = runner
         return this
     }
 
-    fun run(userCount: Int, duration: Duration): ScenarioStatistics {
-        runner.runUsers(userCount, duration) {
+    fun run(userCount: Int, duration: Duration): ScenarioSummary {
+        val summary = runner.runUsers(userCount, duration) {
```

Comment:
You can make the notation more "Kotlinish" here and use `also` scoping function like

```Kotlin
return runner.runUsers(...) { ... }
          .also { StdOutStatisticsPrinter().print(it) } 
```

Reply:
done